### DISCUSSION TITLE
Delay PR creation

### DISCUSSION THREAD

**Discussion by ychulovskyy** [renovatebot/renovate#18092] [Request Help]
Assume we have a Java project with continuous deployment.   
It has GitLab build pipeline that run a few jobs after each merge to main branch (e.g. bump project version and tag git commit).  
Once we merge Renovate Merge Request it creates new Merge Request immediately.  
Which becomes outdated in a minute because another job will make a commit in a minute after merge to main branch.  

Is there possibility to delay Renovate MR creation after merge? E.g. for 3 minutes

**Comment by rarkins** [comment]
There's no functionality to support this capability

**Reply by silouanwright** [reply]
@rarkins could you consider adding this? It would be highly useful.
